{"success": false, "error": "Traceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 544, in _run_solc\n    ret: Dict = json.loads(stdout)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/__init__.py\", line 357, in loads\n    return _default_decoder.decode(s)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 355, in raw_decode\n    raise JSONDecodeError(\"Expecting value\", s, err.value) from None\njson.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 826, in main_impl\n    ) = process_all(filename, args, detector_classes, printer_classes)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 86, in process_all\n    compilations = compile_all(target, **vars(args))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 637, in compile_all\n    compilations.append(CryticCompile(target, **kwargs))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 117, in __init__\n    self._compile(**kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 548, in _compile\n    self._platform.compile(self, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 154, in compile\n    targets_json = _get_targets_json(compilation_unit, self._target, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 276, in _get_targets_json\n    return _run_solc(\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 548, in _run_solc\n    raise InvalidCompilation(f\"Invalid solc compilation {stderr}\")\ncrytic_compile.platform.exceptions.InvalidCompilation: Invalid solc compilation Error: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/84/2.IERC20.sol\" not found: File not found.\n --> /Users/giacomoibba/Desktop/SmarthER/mainnet/84/843cF005B7E96666a268d9d59a7b71278dEbf6DF_GAMETIME.sol:8:1:\n  |\n8 | import \"./2.IERC20.sol\";\n  | ^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/84/3.Ownable.sol\" not found: File not found.\n --> /Users/giacomoibba/Desktop/SmarthER/mainnet/84/843cF005B7E96666a268d9d59a7b71278dEbf6DF_GAMETIME.sol:9:1:\n  |\n9 | import \"./3.Ownable.sol\";\n  | ^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/84/4.Pausable.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/84/843cF005B7E96666a268d9d59a7b71278dEbf6DF_GAMETIME.sol:10:1:\n   |\n10 | import \"./4.Pausable.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/84/5.IERC721.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/84/843cF005B7E96666a268d9d59a7b71278dEbf6DF_GAMETIME.sol:11:1:\n   |\n11 | import \"./5.IERC721.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/84/6.VRFV2WrapperConsumerBase.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/84/843cF005B7E96666a268d9d59a7b71278dEbf6DF_GAMETIME.sol:12:1:\n   |\n12 | import \"./6.VRFV2WrapperConsumerBase.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/84/7.ConfirmedOwner.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/84/843cF005B7E96666a268d9d59a7b71278dEbf6DF_GAMETIME.sol:13:1:\n   |\n13 | import \"./7.ConfirmedOwner.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/84/8.IERC721Receiver.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/84/843cF005B7E96666a268d9d59a7b71278dEbf6DF_GAMETIME.sol:14:1:\n   |\n14 | import \"./8.IERC721Receiver.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/SmarthER/mainnet/84/9.AccessControl.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/SmarthER/mainnet/84/843cF005B7E96666a268d9d59a7b71278dEbf6DF_GAMETIME.sol:15:1:\n   |\n15 | import \"./9.AccessControl.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\n\n", "results": {}}
